### PR TITLE
Add tests for QueryCompletedEventScriber and SlackBot

### PR DESCRIPTION
**Pull request by Yaliang**
Added the thrift definition file into code base so that the Travis will run the tests.
Added tests for QueryCompletedEventScriber
Added tests for SlackBot

### CODE REVIEW

**Comment by Yaliang** [comment]
Ready for reviewing

File: twitter-eventlistener-plugin/src/test/java/com/twitter/presto/plugin/eventlistener/slack/SlackResource.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Objects.requireNonNull;
+
+@Path("/api")
+public class SlackResource
```
Comment (luohao) [inline twitter-eventlistener-plugin/src/test/java/com/twitter/presto/plugin/eventlistener/slack/SlackResource.java]: Should we name it `TestingSlackResource`?

Comment (Yaliang) [reply twitter-eventlistener-plugin/src/test/java/com/twitter/presto/plugin/eventlistener/slack/SlackResource.java]: Done

**Comment by Yaliang** [comment]
Ready for reviewing :)

**Comment by luohao** [comment]
Merged, thanks!

**Pull request merged**